packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
```

Comment:
You need to update the version here to match what's being published.

Reply:
Done.